### ISSUE TITLE
New submissions for Wed,  9 Mar 22

### ISSUE THREAD

**Issue by imjunwei** [imjunwei/ocr-daily-paper#149]
### Language Matters: A Weakly Supervised Pre-training Approach for Scene  Text Detection and Spotting
 - **Authors:** Chuhui Xue, Yu Hao, Shijian Lu, Philip Torr, Song Bai
 - **Subjects:** Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2203.03911
 - **Pdf link:** https://arxiv.org/pdf/2203.03911
 - **Abstract**
 Recently, Vision-Language Pre-training (VLP) techniques have greatly benefited various vision-language tasks by jointly learning visual and textual representations, which intuitively helps in Optical Character Recognition (OCR) tasks due to the rich visual and textual information in scene text images. However, these methods cannot well cope with OCR tasks because of the difficulty in both instance-level text encoding and image-text pair acquisition (i.e. images and captured texts in them). This paper presents a weakly supervised pre-training method that can acquire effective scene text representations by jointly learning and aligning visual and textual information. Our network consists of an image encoder and a character-aware text encoder that extract visual and textual features, respectively, as well as a visual-textual decoder that models the interaction among textual and visual features for learning effective scene text representations. With the learning of textual features, the pre-trained model can attend texts in images well with character awareness. Besides, these designs enable the learning from weakly annotated texts (i.e. partial texts in images without text bounding boxes) which mitigates the data annotation constraint greatly. Experiments over the weakly annotated images in ICDAR2019-LSVT show that our pre-trained model improves F-score by +2.5% and +4.8% while transferring its weights to other text detection and spotting networks, respectively. In addition, the proposed method outperforms existing pre-training techniques consistently across multiple public datasets (e.g., +3.2% and +1.3% for Total-Text and CTW1500).
### YONO: Modeling Multiple Heterogeneous Neural Networks on  Microcontrollers
 - **Authors:** Young D. Kwon, Jagmohan Chauhan, Cecilia Mascolo
 - **Subjects:** Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2203.03794
 - **Pdf link:** https://arxiv.org/pdf/2203.03794
 - **Abstract**
 With the advancement of Deep Neural Networks (DNN) and large amounts of sensor data from Internet of Things (IoT) systems, the research community has worked to reduce the computational and resource demands of DNN to compute on low-resourced microcontrollers (MCUs). However, most of the current work in embedded deep learning focuses on solving a single task efficiently, while the multi-tasking nature and applications of IoT devices demand systems that can handle a diverse range of tasks (activity, voice, and context recognition) with input from a variety of sensors, simultaneously. In this paper, we propose YONO, a product quantization (PQ) based approach that compresses multiple heterogeneous models and enables in-memory model execution and switching for dissimilar multi-task learning on MCUs. We first adopt PQ to learn codebooks that store weights of different models. Also, we propose a novel network optimization and heuristics to maximize the compression rate and minimize the accuracy loss. Then, we develop an online component of YONO for efficient model execution and switching between multiple tasks on an MCU at run time without relying on an external storage device. YONO shows remarkable performance as it can compress multiple heterogeneous models with negligible or no loss of accuracy up to 12.37$\times$. Besides, YONO's online component enables an efficient execution (latency of 16-159 ms per operation) and reduces model loading/switching latency and energy consumption by 93.3-94.5% and 93.9-95.0%, respectively, compared to external storage access. Interestingly, YONO can compress various architectures trained with datasets that were not shown during YONO's offline codebook learning phase showing the generalizability of our method. To summarize, YONO shows great potential and opens further doors to enable multi-task learning systems on extremely resource-constrained devices.
### Exploring Fairness in District-based Multi-party Elections under  different Voting Rules using Stochastic Simulations
 - **Authors:** Adway Mitra
 - **Subjects:** Computers and Society (cs.CY); Multiagent Systems (cs.MA); Social and Information Networks (cs.SI)
 - **Arxiv link:** https://arxiv.org/abs/2203.03720
 - **Pdf link:** https://arxiv.org/pdf/2203.03720
 - **Abstract**
 Many democratic societies use district-based elections, where the region under consideration is geographically divided into districts and a representative is chosen for each district based on the preferences of the electors who reside there. These representatives belong to political parties, and the executive powers are acquired by that party which has a majority of the elected district representatives. In most systems, each elector can express preference for one candidate, though they may have a complete or partial ranking of the candidates/parties. We show that this can lead to situations where many electors are dissatisfied with the election results, which is not desirable in a democracy. The results may be biased towards the supporters of a particular party, and against others. Inspired by current literature on fairness of Machine Learning algorithms, we define measures of fairness to quantify the satisfaction of electors, irrespective of their political choices. We also consider alternative election policies using concepts of voting rules and rank aggregation, to enable voters to express their detailed preferences without making the electoral process cumbersome or opaque. We then evaluate these policies using the aforementioned fairness measures with the help of Monte Carlo simulations. Such simulations are obtained using a proposed stochastic model for election simulation, that takes into account community identities of electors and its role in influencing their residence and political preferences. We show that this model can simulate actual multi-party elections in India. Through extensive simulations, we find that allowing voters to provide 2 preferences reduces the disparity between supporters of different parties in terms of the election result.
### A New Era: Intelligent Tutoring Systems Will Transform Online Learning  for Millions
 - **Authors:** Francois St-Hilaire, Dung Do Vu, Antoine Frau, Nathan Burns, Farid Faraji, Joseph Potochny, Stephane Robert, Arnaud Roussel, Selene Zheng, Taylor Glazier, Junfel Vincent Romano, Robert Belfer, Muhammad Shayan, Ariella Smofsky, Tommy Delarosbil, Seulmin Ahn, Simon Eden-Walker, Kritika Sony, Ansona Onyi Ching, Sabina Elkins, Anush Stepanyan, Adela Matajova, Victor Chen, Hossein Sahraei, Robert Larson, Nadia Markova, Andrew Barkett, Laurent Charlin, Yoshua Bengio, Iulian Vlad Serban, Ekaterina Kochmar
 - **Subjects:** Computers and Society (cs.CY); Artificial Intelligence (cs.AI); Human-Computer Interaction (cs.HC); Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2203.03724
 - **Pdf link:** https://arxiv.org/pdf/2203.03724
 - **Abstract**
 Despite artificial intelligence (AI) having transformed major aspects of our society, less than a fraction of its potential has been explored, let alone deployed, for education. AI-powered learning can provide millions of learners with a highly personalized, active and practical learning experience, which is key to successful learning. This is especially relevant in the context of online learning platforms. In this paper, we present the results of a comparative head-to-head study on learning outcomes for two popular online learning platforms (n=199 participants): A MOOC platform following a traditional model delivering content using lecture videos and multiple-choice quizzes, and the Korbit learning platform providing a highly personalized, active and practical learning experience. We observe a huge and statistically significant increase in the learning outcomes, with students on the Korbit platform providing full feedback resulting in higher course completion rates and achieving learning gains 2 to 2.5 times higher than both students on the MOOC platform and students in a control group who don't receive personalized feedback on the Korbit platform. The results demonstrate the tremendous impact that can be achieved with a personalized, active learning AI-powered system. Making this technology and learning experience available to millions of learners around the world will represent a significant leap forward towards the democratization of education.
### Language Matters: A Weakly Supervised Pre-training Approach for Scene  Text Detection and Spotting
 - **Authors:** Chuhui Xue, Yu Hao, Shijian Lu, Philip Torr, Song Bai
 - **Subjects:** Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2203.03911
 - **Pdf link:** https://arxiv.org/pdf/2203.03911
 - **Abstract**
 Recently, Vision-Language Pre-training (VLP) techniques have greatly benefited various vision-language tasks by jointly learning visual and textual representations, which intuitively helps in Optical Character Recognition (OCR) tasks due to the rich visual and textual information in scene text images. However, these methods cannot well cope with OCR tasks because of the difficulty in both instance-level text encoding and image-text pair acquisition (i.e. images and captured texts in them). This paper presents a weakly supervised pre-training method that can acquire effective scene text representations by jointly learning and aligning visual and textual information. Our network consists of an image encoder and a character-aware text encoder that extract visual and textual features, respectively, as well as a visual-textual decoder that models the interaction among textual and visual features for learning effective scene text representations. With the learning of textual features, the pre-trained model can attend texts in images well with character awareness. Besides, these designs enable the learning from weakly annotated texts (i.e. partial texts in images without text bounding boxes) which mitigates the data annotation constraint greatly. Experiments over the weakly annotated images in ICDAR2019-LSVT show that our pre-trained model improves F-score by +2.5% and +4.8% while transferring its weights to other text detection and spotting networks, respectively. In addition, the proposed method outperforms existing pre-training techniques consistently across multiple public datasets (e.g., +3.2% and +1.3% for Total-Text and CTW1500).
### Foundations for Grassroots Democratic Metaverse
 - **Authors:** Nimrod Talmon, Ehud Shapiro
 - **Subjects:** Computers and Society (cs.CY); Artificial Intelligence (cs.AI); Distributed, Parallel, and Cluster Computing (cs.DC); Multiagent Systems (cs.MA); Social and Information Networks (cs.SI)
 - **Arxiv link:** https://arxiv.org/abs/2203.04090
 - **Pdf link:** https://arxiv.org/pdf/2203.04090
 - **Abstract**
 While the physical lives of many of us are in democracies (one person, one vote - e.g., the EU and the US), our digital lives are mostly in autocracies (one person, all votes - e.g., Facebook). Cryptocurrencies promise liberation but stop short, at plutocracy (one coin, one vote). What would it take for us to live in a digital democracy? This paper offers a vision, a theoretical framework, and an architecture for a grassroots network of autonomous, people-owned, people-operated, and people-governed digital communities, namely a grassroots democratic metaverse. It also charts a roadmap towards realizing it, and identifies unexplored territory for MAS research.
### A Preliminary Study on Aging Examining Online Handwriting
 - **Authors:** Marcos Faundez-Zanuy, Enric Sesa-Nogueras, Josep Roure-Alcobé, Anna Esposito, Jiri Mekyska, Karmele López-de-Ipiña
 - **Subjects:** Human-Computer Interaction (cs.HC); Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2203.03933
 - **Pdf link:** https://arxiv.org/pdf/2203.03933
 - **Abstract**
 In order to develop infocommunications devices so that the capabilities of the human brain may interact with the capabilities of any artificially cognitive system a deeper knowledge of aging is necessary. Especially if society does not want to exclude elder people and wants to develop automatic systems able to help and improve the quality of life of this group of population, healthy individuals as well as those with cognitive decline or other pathologies. This paper tries to establish the variations in handwriting tasks with the goal to obtain a better knowledge about aging. We present the correlation results between several parameters extracted from online handwriting and the age of the writers. It is based on BIOSECURID database, which consists of 400 people that provided several biometric traits, including online handwriting. The main idea is to identify those parameters that are more stable and those more age dependent. One challenging topic for disease diagnose is the differentiation between healthy and pathological aging. For this purpose, it is necessary to be aware of handwriting parameters that are, in general, not affected by aging and those who experiment changes, increase or decrease their values, because of it. This paper contributes to this research line analyzing a selected set of online handwriting parameters provided by a healthy group of population aged from 18 to 70 years. Preliminary results show that these parameters are not affected by aging and therefore, changes in their values can only be attributed to motor or cognitive disorders.
### Language Matters: A Weakly Supervised Pre-training Approach for Scene  Text Detection and Spotting
 - **Authors:** Chuhui Xue, Yu Hao, Shijian Lu, Philip Torr, Song Bai
 - **Subjects:** Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2203.03911
 - **Pdf link:** https://arxiv.org/pdf/2203.03911
 - **Abstract**
 Recently, Vision-Language Pre-training (VLP) techniques have greatly benefited various vision-language tasks by jointly learning visual and textual representations, which intuitively helps in Optical Character Recognition (OCR) tasks due to the rich visual and textual information in scene text images. However, these methods cannot well cope with OCR tasks because of the difficulty in both instance-level text encoding and image-text pair acquisition (i.e. images and captured texts in them). This paper presents a weakly supervised pre-training method that can acquire effective scene text representations by jointly learning and aligning visual and textual information. Our network consists of an image encoder and a character-aware text encoder that extract visual and textual features, respectively, as well as a visual-textual decoder that models the interaction among textual and visual features for learning effective scene text representations. With the learning of textual features, the pre-trained model can attend texts in images well with character awareness. Besides, these designs enable the learning from weakly annotated texts (i.e. partial texts in images without text bounding boxes) which mitigates the data annotation constraint greatly. Experiments over the weakly annotated images in ICDAR2019-LSVT show that our pre-trained model improves F-score by +2.5% and +4.8% while transferring its weights to other text detection and spotting networks, respectively. In addition, the proposed method outperforms existing pre-training techniques consistently across multiple public datasets (e.g., +3.2% and +1.3% for Total-Text and CTW1500).
## Keyword: Text Segmentation
There is no result 
## Keyword: TextSpotter
There is no result 
## Keyword: Text Spotting
There is no result
